api/openapi/v1/system-agent.yaml

```diff
@@ -32,27 +34,56 @@ paths:
   /v1/metrics/{services}:
     get:
       description: Fetch the operating performance metrics of the specified EdgeX
-        services by their unique names - returning null if none is found for the service.
-        HTTP 500 for unknown or unanticipated issues
+        unique names - returning an error message if a problem is encountered for a
+        given service. And returning HTTP 500 for unknown or unanticipated issues
       parameters:
       - name: services
         in: path
-        description: EdgeX micro service names (the unique name of each service comma
-          delimited)
+        description: EdgeX micro service names (the unique name of each service,
```

Comment:
Please avoid the use of parenthesis as described on the `config/{services}` endpoint.